pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -865,6 +865,12 @@ type ServiceInstanceSpec struct {
 	// been made to the secrets from which the parameters are sourced.
 	// +optional
 	UpdateRequests int64 `json:"updateRequests"`
+
+	// DeletionRequested provides a mechanism for requesting a delete without
+	// performing a direct delete (which cannot be undone in cases where the
+	// delete is rejected by the broker). This is the recommended way to
+	// delete, to avoid resources becoming stuck in the deletion phase.
+	DeletionRequested metav1.Time `json:"deletionRequested,omitempty"`
```

Comment:
Is this still the right name now that it's not a bool?

Reply:
nope